showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+
+Based on the report templates provided by the ECDC, reports were filled in automatically  based on the sequencing results. These reports included variants of concern (VoCs), specific set of mutations or deletions defined by the ECDC that changed over time. Similar reports were also generated for the Health Board to consult the Government.
+
+## Dashboards
+
+### Description of the closed dashboard
+The data table of all the samples was made available at the access restricted website for the Health Board employees and project participants to provide epidemiological and virological consultations. 
+
+The database included all metadata provided for the samples, sequencing outcomes and was interactive providing filtering and sorting among other customisation options. In addition data could be downloaded for further custom analysis. The closed website of the project dashboard included also different interactive plots that allowed the users to pick the variables to be plotted (e.g. Ct values against vaccination status, county against lineages), but also lineage overview where individual lineages could be included or dismissed, time frame to be selected. 
+
+In addition, for each sequenced plate mutation plots and Ct value plots were implemented to carry out visual inspection as part of the quality control of the sequencing outcomes. In all the dashboards, users could select if they wanted to include data only sequenced in Estonia, by ECDC or both. 
+
+### Public dashboards
+In addition to the access restricted dashboards, public data on the clades, percentage of all positive samples in Estonia been sequenced and clade distribution across counties, regions and age groups were made available and regularly updated at the [Estonian COVID19Dataportal](https://covid19dataportal.ee/genomics_transcriptomics/).  In order to illustrate the virus evolution a public [Nextstrain Auspiece](https://auspice.biit.cs.ut.ee/ncov/est) instance was set up. In regular time intervals, all samples were rerun to obtain the latest lineage annotations.
+
+## Data brokering to ENA 
+
+To upload data to [European Nucleotide Archive (ENA)](https://www.ebi.ac.uk/ena/browser/) we registered our research [projects at the ENA](https://www.ebi.ac.uk/ena/browser/text-search?query=korogeno-est) following the [guideline](https://ena-docs.readthedocs.io/en/latest/submit/study.html). At the beginning we created individual projects for a data batch but later joined many under one. The Data broker generated ENA metadata files using ENA virus pathogen reporting standard checklist ERC000033 filling out mandatory and recommended fields ([checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000033), [metadata template](https://github.com/ELIXIR-Belgium/ENA-metadata-templates)).  The preparatory work for data brokering to ENA was done by a dedicated Galaxy workflow. A typical workflow of extraction of Galaxy results to the metadata file and metadata ERC000033 file generation took about 1h. 
```

Comment:
```suggestion
To upload data to the [European Nucleotide Archive (ENA)](https://www.ebi.ac.uk/ena/browser/) we registered our research [projects at the ENA](https://www.ebi.ac.uk/ena/browser/text-search?query=korogeno-est) following the [guidelines](https://ena-docs.readthedocs.io/en/latest/submit/study.html). At the beginning, individual projects were created for each data batch. Later on, multiple batches of data were joined under a same project. The Data brokering tool generated ENA metadata files following the ENA virus pathogen reporting standard checklist [ERC000033]([checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000033), and using the equivalent [spreadsheet metadata template](https://github.com/ELIXIR-Belgium/ENA-metadata-templates)).  The preparatory work for data brokering to ENA was done by a dedicated Galaxy workflow. A typical workflow from extraction of Galaxy results data and metadata submission took about 1 hour. 
```

> A typical workflow from extraction of Galaxy results data and metadata submission took about 1 hour. 

For the last sentence, is the alteration OK and can a number of Samples be added to it? So that the reader can have an idea of how many Samples one would get for the 1hour of work time invested.

Reply:
1h of data brokering  per plate, each plate has 94 samples.

Reply:
Data broker didn't use a tool to generate ENA metadata files. Broker made them themself, it also gave time to spot any previous inconsistencies, find mistakes and workflow bugs.

last sentence is okay.